redis/redis.go

```diff
@@ -146,59 +146,114 @@ func (c *Client) PublishEnabled(state *EnabledMessage) {
 		})
 
 		if mErr == nil {
-			c.client.Publish(c.ctx, SyncChannelName, binMsg)
+			c.client.Do(ctx,
+				c.client.B().Publish().
+					Channel(SyncChannelName).
+					Message(rueidis.BinaryString(binMsg)).
+					Build())
 		}
 	}
 }
 
 // GetRedisCache reads the redis cache and publish it to the channel
```

Comment:
Small grammar fix since you're already changing the function:
```suggestion
// GetRedisCache reads the redis cache and publishes it to the channel
```